exercises/react/src/react.h

```diff
@@ -0,0 +1,29 @@
+#ifndef REACT_H
+#define REACT_H
+
+struct reactor;
+struct cell;
+
+typedef int (*compute1) (int);
+typedef int (*compute2) (int, int);
+
+struct reactor *create_reactor();
+// destroy_reactor should free all cells created under that reactor.
```

Comment:
Oh, since I only replaced `free_reactor` with `destroy_reactor`, this sentence still says "free all cells". Perhaps it should say "destroy all cells" instead?

Of course, the test doesn't enforce such a thing, so there is no force behind that "should". I believe (valgrind) the example does. But for submissions to the website there's no guarantee anyone ran valgrind.